src/coreimage.cs

```diff
@@ -463,7 +463,7 @@ interface CIContext_ImageRepresentation {
 		bool WriteJpegRepresentation (CIImage image, NSUrl url, CGColorSpace colorSpace, CIImageRepresentationOptions options, [NullAllowed] out NSError error);
 #endif
 
-		[iOS (11,0)][TV (11,0)][Mac (10,13)]
+		[iOS (11,0)][TV (11,0)][Mac (10,13,4)]
```

Comment:
same
if you have this working in 10.13.[2|3] (whatever you're using) then let's keep our original